drivers/soundwire/intel.c

```diff
@@ -1589,15 +1589,16 @@ static int intel_suspend_runtime(struct device *dev)
 		intel_shim_wake(sdw, false);
 	} else if (clock_stop_quirks & SDW_INTEL_CLK_STOP_BUS_RESET ||
 		!clock_stop_quirks) {
-		ret = sdw_cdns_clock_stop(cdns, true);
+
+		ret = sdw_cdns_enable_interrupt(cdns, false);
```

Comment:
There are sdw transmission in the sdw_cdns_clock_stop() function, and interrupt is required. So we can't disable interrupt before calling sdw_cdns_clock_stop().
```
sdw_cdns_clock_stop() -> sdw_bus_prep_clk_stop() -> sdw_slave_clk_stop_callback() 
                                                 -> sdw_slave_clk_stop_prepare()
```

Reply:
ok, that's right